dev/scripts/main.js

```diff
@@ -218,13 +241,33 @@ bikeApp.travelTimeDifference = function (userTime, distanceDuration) {
 		//if user time input is less than trip destination time, prompt alert
 		alert(`You have selected ${bikeApp.time} hours for your trip, but it will take you ${ Math.floor(distanceDurationMinutes) } minutes to get to your destination. Please adjust your trip time`);
 	} else {
-		confirm(`It will take you ${ Math.floor(distanceDurationMinutes) } minutes to travel from station A to station B.`);
+		bikeApp.displayResults(distanceDurationMinutes); 
 	}
```

Comment:
things to do: 
1. store bikeApp.getDistanceDuration api call in promise
`
bikeApp.getDistanceDuration = function(stationOrigin, stationDestination) {
	var distanceService = new google.maps.DistanceMatrixService();
	distanceService.getDistanceMatrix({
	    origins: [stationOrigin],
	    destinations: [stationDestination],
	    travelMode: google.maps.TravelMode.BICYCLING,
	    unitSystem: google.maps.UnitSystem.METRIC,
	    durationInTraffic: true

	}, // this will be the end of promise
// $.when run this function
	function (response, status) {
	    if (status !== google.maps.DistanceMatrixStatus.OK) {
	        console.log('Error:', status);
	    } else {
	    	const stationTravelTime = response.rows[0].elements[0].duration.value; 
//might have to connect 2 promises together to run next api call 
bikeApp.getDistanceDurationRoundTrip = function(stationOrigin, userDestination) {
 +	var distanceService = new google.maps.DistanceMatrixService();
 +	distanceService.getDistanceMatrix({
 +	    origins: [stationOrigin],
 +	    destinations: [userDestination],
 +	    travelMode: google.maps.TravelMode.BICYCLING,
 +	    unitSystem: google.maps.UnitSystem.METRIC,
 +	    durationInTraffic: true
 +
 +	},
 +	function (response, status) {
 +	    if (status !== google.maps.DistanceMatrixStatus.OK) {
 +	        console.log('Error:', status);
 +	    } else {
 +	    	bikeApp.roundTripTravelTime = Math.floor(((response.rows[0].elements[0].duration.value) / 60) * 2);
// only then AFTER when the two api calls have been made run this function:
   			bikeApp.travelTimeDifference(bikeApp.time, stationTravelTime);
	    }
	});
}`
This should be the fix to our issue